ChegeBryan/politico
# #163670698 Update README.md

### What does this PR do?

Updates [README](https://github.com/ChegeBryan/politico/blob/ch-update-gh-pages-readme-163670698/README.md)

#### Description of Task to be completed?


- [x] Add UI screenshot and link.
- [x] website up badge
- [x] License badge
- [x] Languages badge
- [x] Top language badge
- [x] Repository size badge
- [x] Andela Cohort badge

#### How should this be manually tested?
clone this at `https://github.com/ChegeBryan/politico.git` and check out to `ch-update-gh-pages-readme-163670698` jump in the `UI` directory and open `README.md` on your Markdown Preview tool.

#### What are the relevant pivotal tracker stories?
[#163670698](https://www.pivotaltracker.com/story/show/163670698)

#### Screenshots
##### README new state
![screencapture-github-chegebryan-politico-tree-ch-update-gh-pages-readme-163670698-2019-02-03-12_49_28](https://user-images.githubusercontent.com/40359451/52175297-9f319700-27b2-11e9-93fe-4ad56d898b68.png)
